CHANGELOG.md

```diff
@@ -10,9 +10,10 @@ Prefix your items with `(Template)` if the change is about the template and not
 - Updated Pipeline Code Coverage Task from V1 to V2.
 - Fixed iOS crash by updating package and configuring the interpreter.
 - Added support for arm64 and x86 cpus in the mobile project.
-- Update Uno packages from 5.2.121 to 5.5.95.
+- Updated Uno packages from 5.2.121 to 5.5.95.
```

Comment:
That's not even the correct version...

Reply:
Tommy-Lee forgot to update the changelog. I'll fix it.